File: TypeScript/Operator/not-equals-operator.ts
function main(): void {
    // Check if two strings are not the same and return a boolean
    console.log(checkTwoItemsAreNotEqual("Hello", "Hello"))
}

main()

// Check if two strings are not the same and return a boolean
function checkTwoItemsAreNotEqual(primary: string, secondary: string): boolean {
    return primary != secondary
}

File: Go/Operator/not-equals-operator.go
package main

import (
	"fmt"
)

func main() {
	// Check if two values are not equal to each other and return a bool.
	fmt.Println(notEqualsOperator(1, 2))
}

// Check if two values are not equal to each other and return a bool.
func notEqualsOperator(primary int, secondary int) bool {
	return primary != secondary
}
